blocks/rich-text/format-toolbar/index.js

```diff
@@ -26,19 +30,25 @@ const FORMATTING_CONTROLS = [
 	{
 		icon: 'editor-bold',
 		title: __( 'Bold' ),
+		shortcut: keyboardShortcut(CONTROL, 'B'),
 		format: 'bold',
 	},
 	{
 		icon: 'editor-italic',
 		title: __( 'Italic' ),
+		shortcut: keyboardShortcut(CONTROL, 'I'),
 		format: 'italic',
 	},
 	{
 		icon: 'editor-strikethrough',
 		title: __( 'Strikethrough' ),
+		shortcut: keyboardShortcut(CONTROL, ALT, 'D'),
```

Comment:
This should be `Shift+Alt` for Windows, this is why I would prefer two functions as described in https://github.com/WordPress/gutenberg/pull/4411/files#r185503403.

Reply:
Drive-by thought: since we already have to adopt a specific DSL for shortcuts — the one from TinyMCE: `editor.shortcuts.add( 'access+a', …` — should `keyboardShortcut` adopt the same one?

```js
shortcut: keyboardShortcut( 'access+a' ),
shortcut: keyboardShortcut( 'meta+k' ),
```

Reply:
I saw that but I didn't like the sort of magic strings in there… but I guess it's already done that way in places. Maybe it could be the underlying function for TinyMCE users but I like the different functions idea (eg `getMetaCombination()`) more going forward.

Reply:
Frustratingly though, it looks like that library uses `'meta'` when it really means `'modifier'`–unless that code wants `Windows + K` to make a link, which I doubt.

Copying their codes would be annoying when we don't plan to expose them, so I'll opt not to 😄

Reply:
Fair enough 😄

Reply:
Also in favour of `getMetaCombination( 'k' )` instead of `keyboardShortcut( 'meta+k' )` btw. :)

Reply:
Actually using `getX` seems weird because it's more about *creating* a string than *getting* something from something else. I am about to push an update patch; was gonna use `keyboardShortcutModifier( 'A' )`, `keyboardShortcutAccess( 'G' )`, etc.

They are all modifier keys though, fair! 😄

Reply:
Feel free to omit `get-` if you prefer! :)

Reply:
`primaryComination( 'k' )` and `accessCombination( 'k' )` 🤔

Reply:
Or `primaryKeyboardComination` or `primaryShortcutCombination` or `primaryKeyboardShortcutCombination` or `primaryKeyboardShortcut` or... 😭